src/main/resources/Macros/JWPlayer.xml

```diff
@@ -311,7 +311,7 @@ require(['jQueryNoConflict', 'jwplayer'], function($, jwplayer) {
       <cache>default</cache>
     </property>
     <property>
-      <code>#xwikicontent .jwplayer {
+      <code>#xwikicontent [id^="jwplayer"] {
```

Comment:
I don't like this solution that much. I'd rather follow the solution described [here](https://support.jwplayer.com/customer/portal/questions/6552946-adding-css-padding-or-margin), i.e. to add a wrapper and to specify the margin for the wrapper.